src/app/send/add-edit.component.html

```diff
@@ -15,45 +15,43 @@
                 </div>
             </div>
-            <div class="box">
+            <div class="box" *ngIf="send.type === sendType.Text">
                 <div class="box-content">
-                    <ng-container *ngIf="!editMode">
-                        <div class="box-content-row box-content-row-radio">
-                            <label class="radio-header">{{'whatTypeOfSend' | i18n}}</label>
-                            <div class="item" *ngFor="let o of typeOptions">
-                                <input type="radio" class="radio" [(ngModel)]="send.type" name="Type_{{o.value}}"
-                                    id="type_{{o.value}}" [value]="o.value" (change)="typeChanged(o)"
-                                    [checked]="send.type === o.value" [disabled]="disableSend">
-                                <label class="unstyled" for="type_{{o.value}}">
-                                    {{o.name}}
-                                </label>
-                            </div>
-                        </div>
-                        <div *ngIf="send.type === sendType.File" class="box-content-row" appBowRow>
-                            <label for="file">{{'file' | i18n}}</label>
-                            <input type="file" id="file" class="form-control-file" name="file" required [disabled]="disableSend">
-                            <div class="subtext">{{'sendFileDesc' | i18n}} {{'maxFileSize' | i18n}}</div>
-                        </div>
-                    </ng-container>
-                    <ng-container *ngIf="editMode && send.type === sendType.File">
-                        <div class="box-content-row" appBoxRow>
-                            <label for="file">{{'file' | i18n}}</label>
-                            <div class="row-main">{{send.file.fileName}} ({{send.file.sizeName}})</div>
-                        </div>
-                    </ng-container>
-                    <ng-container *ngIf="send.type === sendType.Text">
-                        <div class="box-content-row" appBoxRow *ngIf="send.type === sendType.Text">
-                            <label for="text">{{'text' | i18n}}</label>
-                            <textarea id="text" name="text" [(ngModel)]="send.text.text" rows="6" [readOnly]="disableSend"></textarea>
-                            <div class="subtext">{{'sendTextDesc' | i18n}}</div>
-                        </div>
-                        <div class="box-content-row box-content-row-checkbox" appBoxRow *ngIf="send.type === sendType.Text">
-                            <label for="hideText">{{'textHiddenByDefault' | i18n}}</label>
-                            <input id="hideText" name="hideText" type="checkbox" [(ngModel)]="send.text.hidden" [disabled]="disableSend">
-                        </div>
-                    </ng-container>
+                    <div class="box-content-row box-content-row-checkbox" appBoxRow *ngIf="send.type === sendType.Text">
```

Comment:
since this is nested in a `<div/>` that already has `*ngIf="send.type === sendType.Text"`, this is redundant here.